storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -926,10 +926,10 @@
     public static final String UI_HTTPS_NEED_CLIENT_AUTH = "ui.https.need.client.auth";
 
     /**
-     * The host that Pacemaker is running on.
+     * The list of servers that Pacemaker is running on.
      */
-    @isString
-    public static final String PACEMAKER_HOST = "pacemaker.host";
+    @isStringList
+    public static final String PACEMAKER_SERVERS = "pacemaker.servers";
```

Comment:
@HeartSaVioR Yep, the 1.x-branch PR will leave pacemaker.host in, but have it deprecated.